app/components/base/form/form-page/form-page.scss

```diff
@@ -43,7 +43,7 @@
 
         > .section-content {
             flex: 1;
-            padding: 0 25px;
+            padding: 0 5px;
```

Comment:
I think 5 looks way to compact. Why did you want to change it?

![image](https://cloud.githubusercontent.com/assets/1031227/25256470/d16453b4-25e5-11e7-9b94-ebafcb655315.png)